modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_util.js

```diff
@@ -234,25 +234,23 @@ AUI.add(
 				A.each(
 					schedulerEvents,
 					function(schedulerEvent) {
-						if (schedulerEvent.isRecurring()) {
+						if (schedulerEvent.isRecurring() || calendarBooking.status === CalendarWorkflow.STATUS_DENIED) {
 							var scheduler = schedulerEvent.get('scheduler');
 
 							var eventRecorder = scheduler.get('eventRecorder');
 
 							eventRecorder.hidePopover();
```

Comment:
I found this `hidePopover()` call strange because I know the popup always close by default. It turned out the call was apparently not necessary, so most of the code that was grouped could be deleted. So, I took this SF off and did this change: https://github.com/brandizzi/liferay-portal/pull/323/commits/a39149f2148a609fde55fcc20cbdfb4a192fc73f If you find any problems, let me know!